sample-compose/app/src/main/java/co/nimblehq/sample/compose/ui/screens/main/second/SecondScreen.kt

```diff
@@ -37,11 +37,15 @@ fun SecondScreen(
 @Composable
 private fun SecondScreenContent(
     id: String,
+    modifier: Modifier = Modifier,
     onUpdateClick: () -> Unit,
```

Comment:
Please cross-check with some config from https://github.com/appKODE/detekt-rules-compose, which forces us to order the `Modifier` with the default value at the end. I think that's a good convention to apply as well 🤔